utils/fix_rules.py

```diff
@@ -562,6 +581,15 @@ def fix_callback(file_contents, yaml_contents):
             cce_pool.remove_cce_from_file(cce)
 
 
+def has_unsorted_prodtype(yaml_file, product_yaml=None):
+    rule = yaml.open_and_macro_expand(yaml_file, product_yaml)
+    if 'prodtype' in rule:
+        sorted_prodtypes = rule['prodtype'].split(',')
```

Comment:
I’d maybe cache this as `split_prodtypes` Variable and then call `sorted(…)` instead of `.sort()`. My 2c.